src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,396 @@
+
+#### Chunk Transformations
+
+Chunk transformations metadata describes the spatial relation between chunks
+of the same sample in an implicit coordinate system.
+
+-   The source frame of reference is the frame of reference of the associated image.
+
+-   The target frame of reference is the implicit coordinate system of the transform.
+
+-   The target frame of reference has the same units as the `PixelSizeUnits` metadata.
+
+-   The chunk transform is described by 2 metadata fields: a transformation matrix and a
+    description of the axis of the matrix.
+
+-   Other transformations should be described in derivatives.
+
+<!---
+The following table will be generated automatically with macros after community review.
+A "manual" table is provided to facilitate the review process.
+-->
+| **Key name**                  | **Requirement level**                               | **Data type**                          | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| ----------------------------- | --------------------------------------------------- | -------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ChunkTransformationMatrix     | RECOMMENDED if `<chunk-index>` is used in filenames | [array][] of [arrays][] of [numbers][] | 3x3 or 4x4 matrix describing spatial chunk transformation, for 2D and 3D respectively (for examples: `[[2, 0, 0], [0, 3, 0], [0, 0, 1]]` in 2D for 2x and 3x scaling along the first and second axis respectively or `[[1, 0, 0, 0], [0, 2, 0, 0], [0, 0, 3, 0], [0, 0, 0, 1]]` in 3D for 2x and 3x scaling along the second and third axis respectively). Note that non-spatial dimensions like time and channel are not included in the transformation matrix. |
```

Comment:
Thnaks @mariehbourget ! I think this is clearer now :) 
I still miss a more direct example to link between this matrixes and the chunk numbers/cases?
Could you maybe add the `ChunkTransformationMatrix` for the chunks of some of the examples in the figure?

Reply:
Yes, I think we could add an example here directly after the table.
I was thinking or re-using the Figure 1b (X and Y translations are not the same length so it's the best example). 
I would give examples of chunk-01 (at the origin) and chunk-05 (with a X and Y translation).
That would look like this:

------
![chunk_figure2](https://user-images.githubusercontent.com/54086142/141365360-28435726-5486-4b87-9b1d-e705129d84fe.png)
Examples of chunk transformation JSON metadata for `chunk-01`and `chunk-05`:

`sub-01_sample-01_chunk-01_<modality_suffix>.json` is at the origin:
```JSON
{
        "PixelSize": [1, 1],
        "PixelSizeUnits": "um",
        "ChunkTransformationMatrix": [[1, 0, 0], [0, 1, 0], [0, 0, 1]],
        "ChunkTransformationMatrixAxis": ["X", "Y"]
}
```
`sub-01_sample-01_chunk-05_<modality_suffix>.json` is translated of `5 um` in `X` and `3 um` in `Y`.
```JSON
{
        "PixelSize": [1, 1],
        "PixelSizeUnits": "um",
        "ChunkTransformationMatrix": [[1, 0, 5], [0, 1, 3], [0, 0, 1]],
        "ChunkTransformationMatrixAxis": ["X", "Y"]
}
```
------
Please let me know if this is what you had in mind, thank you!

Reply:
That's indeed SUPER useful!! Thanks @mariehbourget!!

Reply:
I see that in the current definition of the `ChunkTransformationMatrix` the examples say: 2x and 3x **scaling,** where as in the figure example it says um **traslation**. How do you know if its a scaling or a traslation?

Reply:
Thank you @guiomar, I'll integrate the figure and example later today :)

> I see that in the current definition of the `ChunkTransformationMatrix` the examples say: 2x and 3x **scaling,** where as in the figure example it says um **traslation**. How do you know if its a scaling or a traslation?

They are affine matrix. In the `ChunkTransformationMatix` example, **2** and **3** are the scaling coefficient, and there is no translation (**0** in the last column):
[[**2**, 0, 0],
[0, **3**, 0],
[0, 0, 1]]

And in the `figure 1b example`, there is not scaling (scaling coefficients are **1**) and translations (**5** and **3**) are in the last column.
[[**1**, 0, **5**],
[0, **1**, **3**],
[0, 0, 1]]

Reply:
Nice! Now I think I fully got it! Thanks for the patience and explanations :)
Adding this examples on translations and scaling with the figure, may help to easily clarify some of these newer concepts, like chunks, that are not present in other neuroimaging modalities

Reply:
Thank you again for the feedback, I added the example.